Jint/Engine.cs

```diff
@@ -53,7 +53,7 @@ public sealed partial class Engine : IDisposable
         internal readonly Dictionary<string, Type?> TypeCache = new();
 
         // cache for already wrapped CLR objects to keep object identity
-        internal readonly ConditionalWeakTable<object, ObjectWrapper> _objectWrapperCache = new();
+        internal readonly ConditionalWeakTable<object, ObjectInstance> _objectWrapperCache = new();
```

Comment:
Like I've said before, the type should probably be `IObjectWrapper` as this is cache for *object wrappers*.

Reply:
Ah, but the current contract is for object instance.. Fine..